apps/src/lib/config/mod.rs

```diff
@@ -44,6 +44,9 @@ pub const FILENAME: &str = "config.toml";
 pub const TENDERMINT_DIR: &str = "tendermint";
 /// Chain-specific Anoma DB. Nested in chain dirs.
 pub const DB_DIR: &str = "db";
+/// Relevant to validator nodes only. If provided, the ledger will monitor for
+/// new Ethereum events via this endpoint rather than via a `geth` subprocess.
+pub const ENV_VAR_ETHEREUM_URL: &str = "ANOMA_ETHEREUM_URL";
```

Comment:
Using an environment variable to pass this was the quickest way to get this functionality working but eventually we need a more consistent way of passing configuration like this and in general (maybe a CLI flag to `namadan ledger run`?).